crates/bitcoin-da/src/helpers/parsers.rs

```diff
@@ -469,7 +469,7 @@ mod batch_proof {
     }
 }
 
-#[cfg(all(test, feature = "native"))]
+#[cfg(feature = "native")]
```

Comment:
since tests/test_helpers.rs is not considered to be a test itself, we are unable to use it (at least i couldn't find a nice way). hence, i just removed it. i can carry this method to tests-utils, or if you have any other idea i am definitely open to suggestions